.azuredevops/policies/branchClassification.yml

```diff
@@ -0,0 +1,19 @@
+name: branch_classification
+resource: repository
+disabled: false
+where:
+configuration:
+  branchClassificationSettings:
+    defaultClassification: nonproduction
+    ruleset:
+      - name: prod-branches
+        branchNames:
+          - main
+          - release-test/*
```

Comment:
Tip: if you want to test the production pipeline, create a duplicate branch named `release-test/...`